<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TiliTapahtumatMoottoriDialog</class>
 <widget class="QDialog" name="TiliTapahtumatMoottoriDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sulje</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoNostot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Nostot</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoLaskut">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Laskut</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoSiirrot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Siirrot</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoNostoSumma">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoNostoTili">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tili</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuNostoSumma1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuNostoTili1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuNostoSumma2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuNostoTili2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuNostoSumma3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuNostoTili3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuNostoSumma4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuNostoTili4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoLaskuSumma">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoLaskuLaskuttaja">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Laskuttaja</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoLaskuMaksettuTilille">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Maksettu tilille</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoSiirrotSumma">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoSiirrotSiirrettyTililta">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Siirretty tililtä</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoSiirrotSiirrettyTilille">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Siirretty tilille</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuLaskuSumma1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuLaskuSumma2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuLaskuTili1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuLaskuTili2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuLaskuLaskuttaja1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>Laskuttaja tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuLaskuLaskuttaja2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>Laskuttaja tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoSumma1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoSumma2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoSumma3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoSumma4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>summa tähän</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoTili1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoTili2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoTili3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoTili4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoVastaanottajanTili1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoVastaanottajanTili2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoVastaanottajanTili3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoVastaanottajanTili4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>tili tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoSiirrotViesti">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Viesti</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoViesti1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>viesti tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoViesti2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>viesti tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoViesti3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>viesti tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHaettuSiirtoViesti4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="text">
    <string>viesti tähän</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/Pictures/BankSimulLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>31</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>31</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>31</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>31</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
